sample/src/main/java/com/patrykandpatryk/vico/sample/ui/ChartPager.kt

```diff
@@ -93,8 +97,11 @@ internal fun ChartPager(
             ) {
                 SwipeHint(
                     currentPage = state.currentValue + 1,
-                    pageCount = itemCount,
-                )
+                    pageCount = itemCount
+                ) { direction: Int ->
```

Comment:
For consistency with the rest of the source code, I’d suggest removing the type annotation here.

```suggestion
                ) { direction ->
```